core/mondoo-github-security.mql.yaml

```diff
@@ -445,8 +568,8 @@ queries:
       // if a .github repo exists, then check if the CODE_OF_CONDUCT.md is there
       if ( github.organization.repositories.one(name == ".github") ) {
         github.organization.repositories.where( name == ".github") {
-          files.one( name.downcase == "code_of_conduct.md")
-        }
+          files.one( name.downcase == "code_of_conduct.md") 
+        } || github.repository.files.one( name.downcase == "code_of_conduct.md")
```

Comment:
a where and or check do not work well together

Reply:
Same response as above ☝️